File: lc2011.js
var finalValueAfterOperations = (operations)=>{
    let n = operations.length;
    let ans = 0;
    for(let i=0;i<n;i++){
        ans = (operations[i]=='++X' || operations[i]=='X++')?ans+1:(operations[i]=='--X' || operations[i]=='X--')?ans-1:ans+0;
    }
    return ans;
};

File: py/lc2011.py
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        n = len(operations)
        res = 0
        for i in range(n):
            if operations[i]=='++X' or operations[i]=='X++':
                res = res+1
            elif operations[i]=='--X' or operations[i]=='X--':
                res = res-1
        return res
